source/release-notes.txt

```diff
@@ -10,6 +10,48 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.20
+--------------
+
+*Released November, 2019*
+
+- New and improved :guilabel:`New Connection` user experience with
+  support for all connection options.
+
+- Added support for:
+
+  - Cloning :guilabel:`Favorites`.
+
+  - Adding, viewing and editing documents in JSON.
+
+  - Querying UUIDs via the :guilabel:`Documents` query bar or in
+    :guilabel:`Aggregations`.
+
+- Added support for the following aggregation pipeline operators:
+
+  - :manual:`$set </reference/operator/aggregation/set/>`
+
+  - :manual:`$unset </reference/operator/aggregation/unset/>`
+
+  - :manual:`$replaceWith </manual/reference/operator/aggregation/replaceWith/>`
+
+- Improved inline documentation for aggregation pipeline arguments.
+
+- Removed ``$limit`` ahead of the ``$count`` stage in the aggregation
```

Comment:
I would link the $limit and $count to the manual documentation as well as you did in the bullet above.